app/models/user_profile.rb

```diff
@@ -2,7 +2,7 @@
 
 class UserProfile < ActiveRecord::Base
   # Associations
-  belongs_to :user
+  belongs_to :user, optional: true
```

Comment:
does the optional true here mean that a user does not has to have a userProfile?

Reply:
Yeah it does, not sure if this is super smart. I did it mostly to get my user factory running and work with the profile association. Do you think this harms us somewhere?

Reply:
Or actually it does mean that a profile does not automatically needs to belong to a user. So there can be a profile without a User

Reply:
I'm still not really convinced about the separation of user and user profile. And introducing an optional association seems to only add to the confusion. Is it somehow possible to change that?